brewblox_tilt/tiltScanner.py

```diff
@@ -218,6 +220,10 @@ def handleData(self, data):
         if cal_sg is not None:
             cal_plato = self.sgToPlato(cal_sg)
 
+            if cal_sg < self.lowerBound or cal_sg > self.upperBound:
```

Comment:
This will only discard values when a SG calibration file is used. When a file isn't use, cal_sg will be `None`. You'd be better operating on `decodedData["sg"]`. Otherwise, looks good.